shared/src/test/diff/gadt/ThisMatching.mls

```diff
@@ -8,6 +8,9 @@ module Dummy {
 }
 //│ ╔══[ERROR] Cannot access `this` during object initialization
 //│ ║  l.7: 	  log(if this is Dummy then "duh!" else "huh?")
+//│ ║       	  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+//│ ╟── The access to `this` is here
+//│ ║  l.7: 	  log(if this is Dummy then "duh!" else "huh?")
```

Comment:
You can check whether the `this` location is included in the first location (there are helper methods for that), in which case you should just output "Cannot access `this` during object initialization" with just the precise location of `this`.